forui/test/src/widgets/resizable/resizable_test.dart

```diff
@@ -25,6 +26,8 @@ void main() {
   }
 
   testWidgets('vertical drag downwards', (tester) async {
+    debugDefaultTargetPlatformOverride = TargetPlatform.linux;
```

Comment:
Just curious, why do we want to target this for linux?

Reply:
No reason really, we can target any desktop platform.

Reply:
Rather, why do we need to target desktop platforms?

Reply:
This was done previously to test the platform specific stuff, but I guess it isn’t needed anymore